website/views/static/press-kit.jade

```diff
@@ -19,7 +19,32 @@ block content
   p= env.t('presskitDownload')
     | &nbsp;
     a.btn.btn-success(href='/presskit/presskit.zip') presskit.zip
-  - var imgs = ['Habitica Text Logo', 'Habitica Logo Promo', 'Habitica Gryphon', 'Dread Drag\'on of Dilatory', 'Market Sample Screen', 'Challenges Sample Screen', 'Equipment Sample Screen', 'Guilds Sample Screen', 'HabiticaPromoThin', 'Laundromancer', 'Tasks Page', 'SnackLess Monster', 'Battling the Ghost Stag', 'Stagnant Dishes Boss', 'Necro-Vice']
+  - var imgs = ['Habitica Promo',
```

Comment:
This isn't the right way to do multi-line variables in Jade. To my knowledge, the template will not compile like this. See http://jade-lang.com/reference/code/